### DISCUSSION TITLE
发送文件至手机，没有响应，但手机发电脑却立刻有响应

### DISCUSSION THREAD

**Discussion by Ji-whereNiji** [localsend/localsend#1742] [L-简体中文]
![屏幕截图 2024-09-06 211230](https://github.com/user-attachments/assets/28cd7773-2a4b-4f10-a7f2-a9bc3a346033)
![屏幕截图 2024-09-06 211050](https://github.com/user-attachments/assets/e9cc3deb-7809-4d94-9fee-7ea5707af79f)
用电脑发送文件至手机，无响应，然后超时
![屏幕截图 2024-09-06 211218](https://github.com/user-attachments/assets/ddefbbbb-dc19-41c3-801e-2be30cb4b194)
用手机发送电脑，秒传，（手机开启存储权限）
![屏幕截图 2024-09-06 211741](https://github.com/user-attachments/assets/d2953ede-5580-4606-a8a6-24136da7a098)
对于这个问题，我尝试改源文件名字
![屏幕截图 2024-09-06 212221](https://github.com/user-attachments/assets/702b5240-268e-4752-823d-cede254713d8)
![屏幕截图 2024-09-06 212349](https://github.com/user-attachments/assets/6b519b56-8ed4-4840-b6a4-fbff7835479e)
没有效果，现在尝试保存默认存储位置的方法（本来就是默认存储位置）
![Screenshot_20240906_212650_org localsend localsend_app](https://github.com/user-attachments/assets/b033ac6c-b5b1-4c93-bfeb-8e34df5d0490)
没有效果，使用故障排除
![屏幕截图 2024-09-06 212946](https://github.com/user-attachments/assets/a35101f9-990c-4138-8c01-0e6142582531)
无效果，修改防火墙
![屏幕截图 2024-09-06 213126](https://github.com/user-attachments/assets/810d5aa8-6b91-42ce-8a88-fc54dca83d9b)
无效果，自建规则
![屏幕截图 2024-09-06 213405](https://github.com/user-attachments/assets/947794e4-0da5-480c-a398-b6cc2929f96c)
仍无效果，手机以授权
![Screenshot_20240906_213548](https://github.com/user-attachments/assets/85ae952b-0126-444d-bff1-68743abdf99c)
![Screenshot_20240906_213741_com android permissioncontroller](https://github.com/user-attachments/assets/89f77753-6e53-4d98-9daf-469c4f75ea55)
我已经穷途末路了

**Comment by dgx768** [comment]
设置 - 服务器 -重启下 试试看

**Reply by warma13** [reply]
确实可以，重启服务器后就可以了

**Comment by lifangcheng666** [comment]
电脑要是开网络代理了也会失败

**Reply by TeresaHWP** [reply]
听你的关了小飞机就好了

**Comment by weiweizhen** [comment]
遇到了同样的问题，请问解决了吗？有大神帮忙吗？

**Comment by weiweizhen** [comment]
有人吗？有人在吗？

**Comment by weiweizhen** [comment]


**Reply by weiweizhen** [reply]
已经解决啦，果然留联系方式是可以解决问题的，感恩。

**Comment by weiweizhen** [comment]
天使给的方法是：关闭代理（手机端关闭VPN,电脑端关闭一切加速器）

**Reply by JIANG-R98** [reply]
我有时候开了代理也会失败，我都是去app设置里重启一次服务器

**Comment by weiweizhen** [comment]
我现在就已经解决了 哈哈 发自我的 iPhone在 2024年11月20日，20:01，江儿 ***@***.***> 写道：﻿
我有时候开了代理也会失败，我都是去app设置里重启一次服务器

**Comment by GYZSSDH** [comment]
同样的问题，手机能传电脑，电脑找不到手机。
我的解决办法是拔了网线重新插上，电脑就能发文件给手机了，我试过改防火墙，关代理，重启app设置里的服务器，确保相同wifi都没用，就去看了下电脑里的高级网络设置，发现链接速度变成了100 (Mbps)，于是我就拔了网线重新插，链接速度恢复成1000 (Mbps)，重试发送文件，成功。
愿能帮助到后来者。
